examples/api/lib/material/date_picker/show_date_picker.1.dart

```diff
@@ -0,0 +1,71 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+/// Flutter code sample for basic [showDatePicker].
+
+void main() => runApp(const DatePickerApp());
+
+class DatePickerApp extends StatelessWidget {
+  const DatePickerApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const MaterialApp(
+      home: DatePickerExample(),
+    );
```

Comment:
A couple of ideas:
- Configuring the Scaffold here would provide a tiny performance boost (as `setState` calls wouldn't rebuild the Scaffold)
- Adding an AppBar here would make this consistent with other example files.

```suggestion
    return MaterialApp(
      home: Scaffold(
        appBar: AppBar(title: const Text('showDatePicker Example')),
        body: const Center(child: DatePickerExample()),
      ),
    );
```